models/Category.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require("mongoose");
+var URLSlug = require("mongoose-slug-generator");
+const mongoosePaginate =require("mongoose-paginate")
+mongoose.plugin(URLSlug);
+const categorySchema = new mongoose.Schema({
+      categoryName: {
+      type: String,
+      required: true,
+      
+    },
+      subCategories:{

```

Comment:
what is the purpose of sub category?

Reply:
it is used for Describe more categories like women Ethic  subcategories sarees , suits , kurtis  etc

Reply:
So how you are planning to relate with the product..in product also you have option for choosing sub categories